inst/ms/aggreCAT.qmd

```diff
@@ -2,13 +2,34 @@
 title: "aggreCAT: an R Package for Mathematically Aggregating Expert judgements"
 format:
     jss-pdf:
-        keep-tex: true
-        shortnames: true
-        x11names: true
+      classoption: shortnames
+      keep-tex: true
+      header-includes:
+        - \usepackage[utf8]{inputenc}
+        - \usepackage{amsmath}
+        - \usepackage{amsfonts}
+        - \usepackage{caption}
+        - \usepackage{booktabs}
+        - \usepackage{longtable}
+        - \usepackage{array}
+        - \usepackage{multirow}
+        - \usepackage{wrapfig}
+        - \usepackage{float}
+        #- \usepackage{colortbl}
+        - \usepackage{pdflscape}
+        - \usepackage{tabu}
+        - \usepackage{threeparttable}
+        - \usepackage{threeparttablex}
+        - \usepackage[normalem]{ulem}
+        - \usepackage{makecell}
+        - \usepackage{xcolor}
+        - \newcommand{\blandscape}{\begin{landscape}}
+        - \newcommand{\elandscape}{\end{landscape}}
+        - \usepackage{underscore}
+        # - \usepackage[authoryear,round]{natbib}
+      shortnames: true
+      x11names: true
     jss-html: default
-preamble: >
-  \usepackage{underscore}
-  \usepackage[T1]{fontenc} 
 author:
     # use this syntax to add text on several lines
     # To add another line, use \AND at the end of the previous one as above
```

Comment:
Couple of author list comments:
- flagging that, depending on when we submit, we may no longer be SEFS (but we should get it in in time to not have to worry about the change)
- @psychtek's and @rgroenewegen's ORCIDs are broken because you've added the full URL and not just the ID so the URL doubles up when rendered
- Charles's ORCID also doesn't work because 🤷. When I test the link it just takes me to the "edit my own profile" page.

Reply:
- When I manually go to Charles' orchid link, there is no information listed other than a name.. Not sure if has been removed. Hence the 'edit my own profile'. I'm just going to remove for now until we are ready to submit.
- Fixed @psychtek and @rgroenewegen orchid ID links.